pkg/sources/gitlab/gitlab_test.go

```diff
@@ -247,3 +249,41 @@ func Test_setProgressCompleteWithRepo_Progress(t *testing.T) {
 		}
 	}
 }
+
+func Test_scanRepos_SetProgressComplete(t *testing.T) {
+	testCases := []struct {
+		name         string
+		repos        []string
+		wantComplete bool
+		wantErr      bool
+	}{
+		{
+			name:         "no repos",
+			wantComplete: true,
+		},
+		{
+			name:    "no repos",
+			repos:   []string{"a"},
+			wantErr: true,
```

Comment:
**Nit (optional):** Maybe `name` should be `"one invalid repo"`? Invalid because it doesn't end in `.git`

Reply:
thanks for catching this. I updated all the other tests as well 🙇